pkg/kube/cluster-init.sh

```diff
@@ -1042,7 +1047,7 @@ if [ ! -f /var/lib/all_components_initialized ]; then
         fi
 
 
-        if [ -f /var/lib/longhorn_initialized ]; then
+        if [ ! -e "$SAVE_KUBE_VAR_LIB_DIR" ]; then
```

Comment:
This part I am entirely sure, do we need to take backup on every upgrade of k3s, now that is OK for single node installs, but do we handle it for cluster setup ? We cannot take backup of cluster config I believe. We need to address this issue by next release or so.